src/openrct2/drawing/Sprite.cpp

```diff
@@ -36,6 +36,75 @@
 using namespace OpenRCT2;
 using namespace OpenRCT2::Ui;
 
+static inline uint32 rctc_to_rct2_index(uint32 image) {
```

Comment:
Braces should go onto their own lines.

Reply:
Forgot that one when cleaning up. Thanks.